services/cubejs/index.js

```diff
@@ -217,13 +216,34 @@ const options = {
       dataSchemaFiles: () => dataSchemaFiles({ dataSourceId, authToken }),
     };
   },
+  preAggregationsSchema: ({ securityContext }) => `pre_aggregations_${securityContext?.dataSourceVersion}`,
   telemetry: false,
-  scheduledRefreshTimer: 600, // TODO: install securityContexts for the refresh
-  orchestratorOptions: {
-    queryCacheOptions: {
-      refreshKeyRenewalThreshold: 45,
-      backgroundRenew: false,
-    },
+  scheduledRefreshTimer: 60,
+  scheduledRefreshContexts,
+  externalDbType: 'cubestore',
+  externalDriverFactory: async () => ServerCore.createDriver('cubestore', {
+    host: 'cubestore',
+    port: 3030
+  }),
+  cacheAndQueueDriver: 'cubestore',
+
+  // sql server
+  pgSqlPort: 5432,
+  sqlPort: 3306,
```

Comment:
does it rely to defaults in `CUBEJS_PG_API_URL` and `CUBEJS_MYSQL_API_URL`?

maybe extract the values to env variables also?